ui/public/index.html

```diff
@@ -6,7 +6,7 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
       name="viewport"
     />
-    <meta name="config" th:content="${config}"/>
+    <meta name="config" th:content="${config}" content='{"topic":"mocktopic","producerPath":"/mockproduce","consumerPath":"/mockconsume"}'/>
```

Comment:
Thymeleaf will replace this content when it renders the page so it won't show up. This is more of a placeholder while we don't have a mock backend that's also doing the templating or it means we don't need to worry about mocking templating as this will achieve the same thing